chacra/async/__init__.py

```diff
@@ -3,7 +3,8 @@
 import celery
 from datetime import timedelta
 from chacra import models
-from chacra.util import infer_arch_directory, repo_paths, makedirs
+from chacra.util import infer_arch_directory, repo_paths, makedirs, get_combined_repos, get_extra_repos
+from chacra import util
```

Comment:
Because it got so bad that I gave up and decided to import `util` and forgot to update :/